enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -0,0 +1,310 @@
+# Improved Support for Managing Operators through Policy
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+  [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Enhance the policy framework with a new kind, `OperatorPolicy`, to allow for better management of
+OLM Operator installations on managed clusters. The controller for this kind would be included in
+the configuration-policy-controller container, on managed clusters. The new controller would better
+handle imperative installation steps for OLM objects, and provide better support for upgrade and
+uninstallation scenarios.
+
+## Motivation
+
+It is currently possible to install OLM Operators on managed clusters with `ConfigurationPolicies`,
+but it can be complex and error-prone. For example, multiple policies are often needed in order to
+manage the different OLM resources such as OperatorGroup, Subscription, InstallPlan, and
+ClusterServiceVersion. Upgrading or uninstalling an operator can also be difficult with
+ConfigurationPolicies, since it is not generally as easy as just updating one value or deleting a
+resource.
```

Comment:
> Upgrading or uninstalling an operator can also be difficult with ConfigurationPolicies, since it is not generally as easy as just updating one value or deleting a resource.

Can you be specific? It should actually be the case for upgrades. Is it because of the InstallPlan approval?

Reply:
In addition to the need of install plan approval, issues we ran into: (may be specific to disconnected env)
- catalog src policy becomes compliant before it's truly ready
- subscription policy with 'AtLatestKnown` takes a while to become non-compliant after catalog src is ready
- after install plan/subscription status is updated, there is still a chance for the csv to be stuck in failed state